src/extension.ts

```diff
@@ -168,11 +168,6 @@ export function deactivate() {
 }
 
 async function ensureRuntimeDependencies(extension: vscode.Extension<object>, logger: Logger): Promise<boolean> {
-    const exists = await util.installFileExists(util.InstallFileType.Lock);
```

Comment:
For posterity, is there a reason we're removing this logic?

Reply:
You're right. I'll re-implement this logic!